src/main/steps/task-list/utils.ts

```diff
@@ -130,6 +130,63 @@ export const getAdoptionCertificateDetailsStatus = (userCase: CaseWithId): Secti
     : SectionStatus.IN_PROGRESS;
 };
 
+export const getBirthFatherDetailsStatus = (userCase: CaseWithId): SectionStatus => {
+  const {
+    birthFathersNameOnCertificate,
+    birthFatherFirstNames,
+    birthFatherLastNames,
+    birthFatherStillAlive,
+    birthFatherUnsureAliveReason,
+    birthFatherNationality,
+    birthFatherAdditionalNationalities,
+    birthFatherOccupation,
+    birthFatherAddressKnown,
+  } = userCase;
+
+  if (!birthFathersNameOnCertificate) {
+    return SectionStatus.NOT_STARTED;
+  }
+
+  const nameIsOnCertificate = birthFathersNameOnCertificate === 'Yes';
+
+  const areNamesComplete = !!(birthFatherFirstNames && birthFatherLastNames);
+
+  const isFatherAlive = birthFatherStillAlive === 'Yes';
+  const unsureAliveComplete = birthFatherStillAlive === 'NotSure' && birthFatherUnsureAliveReason;
+  const notAlive = birthFatherStillAlive === 'No';
+  const nationalityComplete = birthFatherNationality?.length
+    ? birthFatherNationality.includes('Not sure')
+      ? // Returns true if 'Not sure' is the only array element, else false
+        birthFatherNationality?.length === 1
+      : birthFatherNationality.includes('Other')
+      ? // Returns true if additional nationality array has elements, else false
+        !!birthFatherAdditionalNationalities?.length
+      : // Doesn't contain 'Other' and has over 1 array element, true
+        true
+    : // Empty array, false
+      false;
+  const lastAddressKnown = birthFatherAddressKnown !== 'No';
+  const isAddressComplete = addressComplete(userCase, FieldPrefix.BIRTH_FATHER);
+
+  return nameIsOnCertificate
+    ? areNamesComplete
+      ? isFatherAlive
+        ? nationalityComplete
+          ? birthFatherOccupation
+            ? lastAddressKnown
+              ? isAddressComplete
+                ? SectionStatus.COMPLETED
+                : SectionStatus.IN_PROGRESS
+              : SectionStatus.COMPLETED
+            : SectionStatus.IN_PROGRESS
+          : SectionStatus.IN_PROGRESS
+        : unsureAliveComplete || notAlive
+        ? SectionStatus.COMPLETED
+        : SectionStatus.IN_PROGRESS
+      : SectionStatus.IN_PROGRESS
+    : SectionStatus.COMPLETED;
```

Comment:
it's not a good practice to nest ternary operator to this level 🌞  probably a good read
https://medium.com/@benlmsc/stop-using-nested-ternary-operators-heres-why-53e7e078e65a